tests/unittests/test_permission_error.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+from unittest import mock
+import gzip
+
+from paramiko import file
+import tap_sftp.client as client
+import tap_sftp.sync as sync
+import paramiko
+
+@mock.patch("tap_sftp.client.SFTPConnection.sftp")
+@mock.patch("tap_sftp.client.LOGGER.info")
+class TestPermissionError(unittest.TestCase):
+
+    def test_no_error(self, mocked_logger, mocked_connect):
+
+        mocked_connect.side_effect = paramiko.SFTPClient
+        mocked_connect.open.side_effect = mock.mock_open()
+
+        conn = client.SFTPConnection("10.0.0.1", "username", port="22")
+
+        try:
+            file_handle = conn.get_file_handle({"filepath": "/root_dir/file.csv.gz", "last_modified": "2020-01-01"})
+        except OSError:
+            # check if logger is called if logger is called in the function 
+            # then error has occurred otherwise not
+            self.assertEquals(0, mocked_logger.call_count)
```

Comment:
This test case will not assert anything in case of positive scenario

Reply:
Removed the try-except block